### ISSUE TITLE
NTR: Skin biopsy: abnormal morphology (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9057]
New term request Skin biopsy: abnormal morphology (f440):   UMLS: 0853474

**Comment by pnrobinson** [comment]
@MickeySegal
the HPO would want to specify the concrete abnormality rather than the mode of visualization -- can we specify what is meant?

**Comment by MickeySegal** [comment]
This was a mixture of 3 types of situations:
1.  Diseases for which a skin biopsy is no longer used
2.  Diseases for which we already used a specific finding for the abnormality
3.  CADASIL, for which skin biopsies are used but we didn't have the specific result.

So I deleted all diseases except CADASIL from this finding and changed this finding to "Skin biopsy: granular osmiophilic material" with explanatory terms "GOM; seen on electron microscopy".  See the Skin Biopsy section of https://www.ncbi.nlm.nih.gov/books/NBK1500/ for details (with a typo in the term, for which I've submitted a correction).  I will use HP:0034379 for this; you got osmiophilic correct; the typo found in GeneReviews is even described in https://wikidiff.com/osmiophilic/osmophilic.
